cdap-docs/developers-manual/source/getting-started/start-stop-cdap.rst

```diff
@@ -33,4 +33,4 @@ UI running at ``http://localhost:9999``, where you can deploy example applicatio
 interact with CDAP. 
 
 Note that in the case of the Docker image, you will need to substitute 
-the Docker VM's IP address for ``localhost`` in the web browser address bar.
\ No newline at end of file
+the Docker VM's IP address for ``localhost`` in the web browser address bar.
```

Comment:
This is again only true for wrapper-based usage. Normal docker usage, with a docker daemon running containers on the localhost will proxy the ports on `0.0.0.0` so localhost definitely will work. If you're running your docker container within a VM, you'll need to replace the IP.